frontend/openapi/apis/AdminApi.ts

```diff
@@ -0,0 +1,244 @@
+// @ts-nocheck
```

Comment:
This file should not be generated - admin endpoints are not planned to be used on frontend

Reply:
I couldnt find an easy way to exclude files from being generated. One possible solution would be if the dev swagger.json only included endpoints that are meant to be used on frontend. (Though im not sure how hard that would be, so I guess the simplest way would be to just not include them in the apiClient instance)

Reply:
If theres no way to exclude it then don't bother I guess